openspec/changes/archive/2026-04-05-workspace-trigram-index/verify-report.md

```
@@ -0,0 +1,135 @@
+# Verification Report
+
+**Change**: workspace-trigram-index
+**Version**: N/A
+
+---
+
+### Completeness
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)